apps/amo/tests/test_views.py

```diff
@@ -390,3 +390,13 @@ def test_contribute_json(self):
         res = self.client.get('/contribute.json')
         eq_(res.status_code, 200)
         eq_(res._headers['content-type'], ('Content-Type', 'application/json'))
+
+
+class TestRobots(amo.tests.TestCase):
+
+    def test_disable_collections(self):
+        """Make sure /en-US/firefox/collections/ gets disabled"""
+        url = reverse('collections.list')
+        response = self.client.get('/robots.txt')
+        eq_(response.status_code, 200)
```

Comment:
Since we're using pytest I think it'd be perfect to update the tests at some point to kick out nose all together. There isn't much used of it anyway except for a few `eq_`, `ok_` and `raises` calls (among others but all have pytest equivalents).